[at-spi2-core] Build error

Package: at-spi2-core:x64-linux -> 2.44.1#2

**Host Environment**

- Host: x64-linux
- Compiler: GNU 13.2.1
-    vcpkg-tool version: 2023-09-15-ac02a9f660977426b8ec6392919fbb1d51b10998
    vcpkg-scripts version: cc97b4536 2023-10-06 (2 days ago)

**To Reproduce**

`vcpkg install wxwidgets`
**Failure logs**

```
-- at-spi2-core currently requires the following libraries from the system package manager:
    libdbus-1-dev
    libxi-dev
    libxtst-dev

These can be installed on Ubuntu systems via apt-get install libdbus-1-dev libxi-dev libxtst-dev
-- Using cached GNOME-at-spi2-core-AT_SPI2_CORE_2_44_1.tar.gz.
-- Cleaning sources at /home/jason/vcpkg/buildtrees/at-spi2-core/src/ORE_2_44_1-fa40e2a5ad.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /home/jason/vcpkg/downloads/GNOME-at-spi2-core-AT_SPI2_CORE_2_44_1.tar.gz
-- Using source at /home/jason/vcpkg/buildtrees/at-spi2-core/src/ORE_2_44_1-fa40e2a5ad.clean
-- Getting CMake variables for x64-linux-dbg
-- Getting CMake variables for x64-linux-rel
-- Configuring x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /usr/bin/python3 /home/jason/vcpkg/installed/x64-linux/tools/meson/meson.py -Dintrospection=no --buildtype plain --backend ninja --wrap-mode nodownload --libdir lib --native /home/jason/vcpkg/buildtrees/at-spi2-core/meson-x64-linux-dbg.log -Ddebug=true --prefix /home/jason/vcpkg/packages/at-spi2-core_x64-linux/debug --includedir ../include -Dcmake_prefix_path=['/home/jason/vcpkg/installed/x64-linux/debug','/home/jason/vcpkg/installed/x64-linux'] /home/jason/vcpkg/buildtrees/at-spi2-core/src/ORE_2_44_1-fa40e2a5ad.clean
    Working Directory: /home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg
    Error code: 1
    See logs for more information:
      /home/jason/vcpkg/buildtrees/at-spi2-core/config-x64-linux-dbg-meson-log.txt.log
      /home/jason/vcpkg/buildtrees/at-spi2-core/config-x64-linux-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_configure_meson.cmake:431 (vcpkg_execute_required_process)
  ports/at-spi2-core/portfile.cmake:13 (vcpkg_configure_meson)
  scripts/ports.cmake:168 (include)



```
<details><summary>/home/jason/vcpkg/buildtrees/at-spi2-core/config-x64-linux-dbg-meson-log.txt.log</summary>

```
Build started at 2023-10-08T19:52:29.238156
Main binary: /usr/bin/python3
Build Options: -Dintrospection=no -Ddebug=true '-Dcmake_prefix_path=['"'"'/home/jason/vcpkg/installed/x64-linux/debug'"'"','"'"'/home/jason/vcpkg/installed/x64-linux'"'"']' -Dprefix=/home/jason/vcpkg/packages/at-spi2-core_x64-linux/debug -Dincludedir=../include -Dlibdir=lib -Dbackend=ninja -Dbuildtype=plain -Dwrap_mode=nodownload '--native-file /home/jason/vcpkg/buildtrees/at-spi2-core/meson-x64-linux-dbg.log'
Python system: Linux
The Meson build system
Version: 0.62.1
Source dir: /home/jason/vcpkg/buildtrees/at-spi2-core/src/ORE_2_44_1-fa40e2a5ad.clean
Build dir: /home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg
Build type: native build
Project name: at-spi2-core
Project version: 2.44.1
Sanity testing C compiler: /usr/lib64/ccache/cc
Is cross compiler: False.
Sanity check compiler command line: /usr/lib64/ccache/cc sanitycheckc.c -o sanitycheckc.exe -fPIC -g -I/home/jason/vcpkg/installed/x64-linux/include -D_FILE_OFFSET_BITS=64 -L/home/jason/vcpkg/installed/x64-linux/debug/lib
Sanity check compile stdout:

-----
Sanity check compile stderr:

-----
Running test binary command: /home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg/meson-private/sanitycheckc.exe
C compiler for the host machine: /usr/lib64/ccache/cc (gcc 13.2.1 "cc (GCC) 13.2.1 20230728 (Red Hat 13.2.1-1)")
C linker for the host machine: /usr/lib64/ccache/cc ld.bfd 2.39-9
Sanity testing C compiler: /usr/lib64/ccache/cc
Is cross compiler: False.
Sanity check compiler command line: /usr/lib64/ccache/cc sanitycheckc.c -o sanitycheckc.exe -D_FILE_OFFSET_BITS=64
Sanity check compile stdout:

-----
Sanity check compile stderr:

-----
Running test binary command: /home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg/meson-private/sanitycheckc.exe
C compiler for the build machine: /usr/lib64/ccache/cc (gcc 13.2.1 "cc (GCC) 13.2.1 20230728 (Red Hat 13.2.1-1)")
C linker for the build machine: /usr/lib64/ccache/cc ld.bfd 2.39-9
Build machine cpu family: x86_64
Build machine cpu: x86_64
Host machine cpu family: x86_64
Host machine cpu: x86_64
Target machine cpu family: x86_64
Target machine cpu: x86_64
Pkg-config binary for 1 is not cached.
Pkg-config binary for 1 specified from cross file, native file, or env var as ['/bin/pkg-config']
Found pkg-config: /bin/pkg-config (1.8.0)
Determining dependency 'dbus-1' with pkg-config executable '/bin/pkg-config'
env[PKG_CONFIG]: /bin/pkg-config
env[PKG_CONFIG_PATH]: /home/jason/vcpkg/installed/x64-linux/debug/lib/pkgconfig/:/home/jason/vcpkg/installed/x64-linux/debug/share/pkgconfig/
Called `/bin/pkg-config --modversion dbus-1` -> 1

CMake binary for 1 is not cached
CMake binary for 1 specified from cross file, native file, or env var as ['/usr/bin/cmake']
Found CMake: /usr/bin/cmake (3.27.6)
Extracting basic cmake information
CMake Toolchain: Calling CMake once to generate the compiler state
Calling CMake (['/usr/bin/cmake']) in /home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg/meson-private/__CMake_compiler_info__ with:
  - "--trace-expand"
  - "--trace-format=json-v1"
  - "--no-warn-unused-cli"
  - "--trace-redirect=cmake_trace.txt"
  - "-G"
  - "Ninja"
  - "-DCMAKE_TOOLCHAIN_FILE=/home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg/meson-private/__CMake_compiler_info__/CMakeMesonTempToolchainFile.cmake"
  - "."
  - "-DCMAKE_PREFIX_PATH=/home/jason/vcpkg/installed/x64-linux/debug;/home/jason/vcpkg/installed/x64-linux"
CMake trace warning: add_executable() non imported executables are not supported
CMake TRACE: /home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg/meson-private/__CMake_compiler_info__/CMakeFiles/CMakeScratch/TryCompile-eFu5El/CMakeLists.txt:20 add_executable(['cmTC_62f94', '/usr/share/cmake/Modules/CMakeCCompilerABI.c'])
CMake trace warning: target_link_options() TARGET cmTC_62f94 not found
CMake TRACE: /home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg/meson-private/__CMake_compiler_info__/CMakeFiles/CMakeScratch/TryCompile-eFu5El/CMakeLists.txt:23 target_link_libraries(['cmTC_62f94', ''])
Try CMake generator: auto
Calling CMake (['/usr/bin/cmake']) in /home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg/meson-private/cmake_dbus-1 with:
  - "--trace-expand"
  - "--trace-format=json-v1"
  - "--no-warn-unused-cli"
  - "--trace-redirect=cmake_trace.txt"
  - "-DCMAKE_TOOLCHAIN_FILE=/home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg/meson-private/cmake_dbus-1/CMakeMesonToolchainFile.cmake"
  - "."
  - "-DCMAKE_PREFIX_PATH=/home/jason/vcpkg/installed/x64-linux/debug;/home/jason/vcpkg/installed/x64-linux"
  -- Module search paths:    ['/', '/home/jason/vcpkg/installed/x64-linux', '/home/jason/vcpkg/installed/x64-linux/debug', '/opt', '/usr', '/usr/local']
  -- CMake root:             /usr/share/cmake
  -- CMake architectures:    []
  -- CMake lib search paths: ['lib', 'lib32', 'lib64', 'libx32', 'share']
Preliminary CMake check failed. Aborting.
Run-time dependency dbus-1 found: NO (tried pkgconfig and cmake)

../src/ORE_2_44_1-fa40e2a5ad.clean/meson.build:52:0: ERROR: Dependency "dbus-1" not found, tried pkgconfig and cmake
```
</details>
<details><summary>/home/jason/vcpkg/buildtrees/at-spi2-core/config-x64-linux-dbg-out.log</summary>

```
WARNING: Recommend using either -Dbuildtype or -Doptimization + -Ddebug. Using both is redundant since they override each other. See: https://mesonbuild.com/Builtin-options.html#build-type-options
The Meson build system
Version: 0.62.1
Source dir: /home/jason/vcpkg/buildtrees/at-spi2-core/src/ORE_2_44_1-fa40e2a5ad.clean
Build dir: /home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg
Build type: native build
Project name: at-spi2-core
Project version: 2.44.1
C compiler for the host machine: /usr/lib64/ccache/cc (gcc 13.2.1 "cc (GCC) 13.2.1 20230728 (Red Hat 13.2.1-1)")
C linker for the host machine: /usr/lib64/ccache/cc ld.bfd 2.39-9
Host machine cpu family: x86_64
Host machine cpu: x86_64
Found pkg-config: /bin/pkg-config (1.8.0)
Found CMake: /usr/bin/cmake (3.27.6)
Run-time dependency dbus-1 found: NO (tried pkgconfig and cmake)

../src/ORE_2_44_1-fa40e2a5ad.clean/meson.build:52:0: ERROR: Dependency "dbus-1" not found, tried pkgconfig and cmake

A full log can be found at /home/jason/vcpkg/buildtrees/at-spi2-core/x64-linux-dbg/meson-logs/meson-log.txt
```
</details>

--- a/ports/at-spi2-atk/portfile.cmake
+++ b/ports/at-spi2-atk/portfile.cmake
@@ -1,7 +1,3 @@
-if(VCPKG_TARGET_IS_LINUX)
-    message(STATUS "${PORT} currently requires the following libraries from the system package manager:\n    libdbus-1\n\nThese can be installed on Ubuntu systems via apt-get install libdbus-1-dev")
-endif()
-
 vcpkg_from_gitlab(
     OUT_SOURCE_PATH SOURCE_PATH
     GITLAB_URL https://gitlab.gnome.org
@@ -22,7 +18,7 @@ vcpkg_copy_pdbs()
 
 vcpkg_fixup_pkgconfig()
 
-file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/COPYING")
 
 if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
     file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/bin" "${CURRENT_PACKAGES_DIR}/debug/bin")

--- a/ports/at-spi2-atk/vcpkg.json
+++ b/ports/at-spi2-atk/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "at-spi2-atk",
   "version": "2.38.0",
+  "port-version": 1,
   "description": "Implementation of the ATK interfaces in terms of the libatspi2 API.",
   "homepage": "https://www.gtk.org/",
   "license": null,
@@ -8,6 +9,10 @@
   "dependencies": [
     "at-spi2-core",
     "atk",
+    {
+      "name": "dbus",
+      "platform": "linux"
+    },
     "libxml2",
     {
       "name": "vcpkg-tool-meson",

--- a/ports/at-spi2-core/portfile.cmake
+++ b/ports/at-spi2-core/portfile.cmake
@@ -1,5 +1,5 @@
 if(VCPKG_TARGET_IS_LINUX)
-    message(STATUS "${PORT} currently requires the following libraries from the system package manager:\n    libdbus-1-dev\n    libxi-dev\n    libxtst-dev\n\nThese can be installed on Ubuntu systems via apt-get install libdbus-1-dev libxi-dev libxtst-dev")
+    message(STATUS "${PORT} currently requires the following libraries from the system package manager:\n    libxi-dev\n    libxtst-dev\n\nThese can be installed on Ubuntu systems via apt-get install libxi-dev libxtst-dev")
 endif()
 
 vcpkg_from_gitlab(
@@ -36,7 +36,7 @@ if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
     )
 endif()
 
-file(INSTALL "${SOURCE_PATH}/COPYING" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}" RENAME copyright)
+vcpkg_install_copyright(FILE_LIST "${SOURCE_PATH}/COPYING")
 
 file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share" "${CURRENT_PACKAGES_DIR}/share/defaults")
 

--- a/ports/at-spi2-core/vcpkg.json
+++ b/ports/at-spi2-core/vcpkg.json
@@ -1,12 +1,16 @@
 {
   "name": "at-spi2-core",
   "version": "2.44.1",
-  "port-version": 2,
+  "port-version": 3,
   "description": "Base DBus XML interfaces for accessibility, the accessibility registry daemon, and atspi library.",
   "homepage": "https://www.gtk.org/",
   "license": null,
   "supports": "linux",
   "dependencies": [
+    {
+      "name": "dbus",
+      "platform": "linux"
+    },
     "glib",
     {
       "name": "glib",

--- a/versions/a-/at-spi2-atk.json
+++ b/versions/a-/at-spi2-atk.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "97535464d9f15ce3bcadaf78ad8031fa3df5eda5",
+      "version": "2.38.0",
+      "port-version": 1
+    },
+    {
       "git-tree": "ebb28a9cdbd06d3185ba2e0cfc1945be23a0a608",
       "version": "2.38.0",
       "port-version": 0

--- a/versions/a-/at-spi2-core.json
+++ b/versions/a-/at-spi2-core.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "5143414003b96ead1ffe40c57c9bf764bf64b0c0",
+      "version": "2.44.1",
+      "port-version": 3
+    },
+    {
       "git-tree": "897df7693a8a1addc3a5ab84efabef89e4cef1d8",
       "version": "2.44.1",
       "port-version": 2

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -286,11 +286,11 @@
     },
     "at-spi2-atk": {
       "baseline": "2.38.0",
-      "port-version": 0
+      "port-version": 1
     },
     "at-spi2-core": {
       "baseline": "2.44.1",
-      "port-version": 2
+      "port-version": 3
     },
     "atk": {
       "baseline": "2.38.0",

